```
Error running mint automation
+ mkdir -p 9918-c4ed1ac/gopath/src/github.com/minio
+ git -C 9918-c4ed1ac/gopath/src/github.com/minio clone --quiet https://github.com/minio/minio.git
+ git -C 9918-c4ed1ac/gopath/src/github.com/minio/minio remote add minio https://github.com/minio/minio.git
+ git -C 9918-c4ed1ac/gopath/src/github.com/minio/minio fetch --quiet minio pull/9918/head:pr9918
+ git -C 9918-c4ed1ac/gopath/src/github.com/minio/minio checkout --quiet pr9918
+ GO111MODULE=on
+ GOPATH=/home/mint-auto/mint-auto/9918-c4ed1ac/gopath
+ make -C 9918-c4ed1ac/gopath/src/github.com/minio/minio --quiet
go: downloading github.com/minio/cli v1.22.0
go: downloading google.golang.org/api v0.5.0
go: downloading github.com/minio/minio-go/v6 v6.0.58-0.20200612001654-a57fec8037ec
go: downloading github.com/Azure/azure-storage-blob-go v0.8.0
go: downloading github.com/json-iterator/go v1.1.10
go: downloading github.com/willf/bloom v2.0.3+incompatible
go: extracting github.com/json-iterator/go v1.1.10
go: extracting github.com/Azure/azure-storage-blob-go v0.8.0
go: downloading github.com/dchest/siphash v1.2.1
go: extracting github.com/dchest/siphash v1.2.1
go: extracting github.com/willf/bloom v2.0.3+incompatible
go: downloading github.com/djherbis/atime v1.0.0
go: extracting github.com/minio/cli v1.22.0
go: downloading gopkg.in/jcmturner/gokrb5.v7 v7.3.0
go: extracting github.com/djherbis/atime v1.0.0
go: downloading github.com/tinylib/msgp v1.1.1
go: extracting github.com/minio/minio-go/v6 v6.0.58-0.20200612001654-a57fec8037ec
go: extracting github.com/tinylib/msgp v1.1.1
go: downloading github.com/willf/bitset v1.1.10
go: downloading github.com/klauspost/pgzip v1.2.1
go: downloading github.com/minio/simdjson-go v0.1.5-0.20200303142138-b17fe061ea37
go: downloading github.com/dgrijalva/jwt-go v3.2.0+incompatible
go: downloading golang.org/x/crypto v0.0.0-20200604202706-70a84ac30bf9
go: extracting github.com/willf/bitset v1.1.10
go: downloading cloud.google.com/go v0.39.0
go: extracting github.com/klauspost/pgzip v1.2.1
go: extracting github.com/dgrijalva/jwt-go v3.2.0+incompatible
go: downloading github.com/bcicen/jstream v0.0.0-20190220045926-16c1f8af81c2
go: downloading github.com/miekg/dns v1.1.8
go: extracting github.com/miekg/dns v1.1.8
go: downloading github.com/cespare/xxhash v1.1.0
go: extracting github.com/cespare/xxhash v1.1.0
go: extracting github.com/bcicen/jstream v0.0.0-20190220045926-16c1f8af81c2
go: downloading github.com/cespare/xxhash/v2 v2.1.1
go: extracting golang.org/x/crypto v0.0.0-20200604202706-70a84ac30bf9
go: extracting github.com/cespare/xxhash/v2 v2.1.1
go: downloading github.com/modern-go/reflect2 v1.0.1
go: downloading github.com/colinmarc/hdfs/v2 v2.1.1
go: extracting github.com/modern-go/reflect2 v1.0.1
go: downloading github.com/cheggaaa/pb v1.0.28
go: extracting github.com/cheggaaa/pb v1.0.28
go: downloading github.com/montanaflynn/stats v0.5.0
go: extracting github.com/minio/simdjson-go v0.1.5-0.20200303142138-b17fe061ea37
go: extracting github.com/montanaflynn/stats v0.5.0
go: downloading github.com/klauspost/reedsolomon v1.9.9
go: extracting github.com/colinmarc/hdfs/v2 v2.1.1
go: extracting cloud.google.com/go v0.39.0
go: downloading github.com/nats-io/stan.go v0.4.5
go: downloading github.com/klauspost/compress v1.10.3
go: extracting github.com/nats-io/stan.go v0.4.5
go: downloading golang.org/x/sys v0.0.0-20200610111108-226ff32320da
go: extracting github.com/klauspost/reedsolomon v1.9.9
go: downloading gopkg.in/ldap.v3 v3.0.3
go: downloading github.com/streadway/amqp v0.0.0-20190404075320-75d898a42a94
go: extracting gopkg.in/ldap.v3 v3.0.3
go: downloading github.com/minio/highwayhash v1.0.0
go: extracting github.com/streadway/amqp v0.0.0-20190404075320-75d898a42a94
go: extracting github.com/minio/highwayhash v1.0.0
go: downloading github.com/prometheus/client_golang v0.9.3
go: downloading github.com/nats-io/nuid v1.0.1
go: extracting github.com/nats-io/nuid v1.0.1
go: downloading golang.org/x/net v0.0.0-20200602114024-627f9648deb9
go: extracting github.com/prometheus/client_golang v0.9.3
go: extracting golang.org/x/sys v0.0.0-20200610111108-226ff32320da
go: downloading github.com/mattn/go-isatty v0.0.8
go: extracting github.com/mattn/go-isatty v0.0.8
go: extracting golang.org/x/net v0.0.0-20200602114024-627f9648deb9
go: downloading google.golang.org/grpc v1.22.0
go: downloading github.com/shirou/gopsutil v2.20.3-0.20200314133625-53cec6b37e6a+incompatible
go: downloading github.com/coreos/etcd v3.3.12+incompatible
go: extracting github.com/shirou/gopsutil v2.20.3-0.20200314133625-53cec6b37e6a+incompatible
go: extracting google.golang.org/grpc v1.22.0
go: downloading github.com/prometheus/common v0.4.0
go: downloading github.com/eclipse/paho.mqtt.golang v1.2.0
go: extracting github.com/eclipse/paho.mqtt.golang v1.2.0
go: downloading github.com/gomodule/redigo v2.0.0+incompatible
go: extracting github.com/prometheus/common v0.4.0
go: extracting github.com/gomodule/redigo v2.0.0+incompatible
go: downloading github.com/Shopify/sarama v1.24.1
go: extracting github.com/Shopify/sarama v1.24.1
go: extracting github.com/coreos/etcd v3.3.12+incompatible
go: downloading github.com/minio/sio v0.2.0
go: extracting github.com/minio/sio v0.2.0
go: downloading gopkg.in/asn1-ber.v1 v1.0.0-20181015200546-f715ec2f112d
go: extracting gopkg.in/asn1-ber.v1 v1.0.0-20181015200546-f715ec2f112d
go: downloading github.com/xdg/scram v0.0.0-20180814205039-7eeb5667e42c
go: extracting github.com/xdg/scram v0.0.0-20180814205039-7eeb5667e42c
go: downloading github.com/inconshreveable/go-update v0.0.0-20160112193335-8152e7eb6ccf
go: extracting github.com/inconshreveable/go-update v0.0.0-20160112193335-8152e7eb6ccf
go: extracting github.com/klauspost/compress v1.10.3
go: downloading github.com/mitchellh/go-homedir v1.1.0
go: downloading github.com/lib/pq v1.1.1
go: extracting github.com/mitchellh/go-homedir v1.1.0
go: downloading github.com/spaolacci/murmur3 v1.1.0
go: extracting github.com/lib/pq v1.1.1
go: downloading github.com/fatih/color v1.7.0
go: extracting github.com/spaolacci/murmur3 v1.1.0
go: downloading google.golang.org/genproto v0.0.0-20190513181449-d00d292a067c
go: extracting github.com/fatih/color v1.7.0
go: downloading github.com/gorilla/rpc v1.2.0
go: extracting github.com/gorilla/rpc v1.2.0
go: downloading github.com/secure-io/sio-go v0.3.0
go: extracting github.com/secure-io/sio-go v0.3.0
go: downloading github.com/elazarl/go-bindata-assetfs v1.0.0
go: extracting github.com/elazarl/go-bindata-assetfs v1.0.0
go: downloading gopkg.in/olivere/elastic.v5 v5.0.80
go: extracting gopkg.in/olivere/elastic.v5 v5.0.80
go: downloading github.com/pkg/errors v0.8.1
go: extracting github.com/pkg/errors v0.8.1
go: downloading github.com/prometheus/client_model v0.0.0-20190129233127-fd36f4220a90
go: extracting github.com/prometheus/client_model v0.0.0-20190129233127-fd36f4220a90
go: downloading github.com/gorilla/mux v1.7.0
go: extracting github.com/gorilla/mux v1.7.0
go: downloading github.com/gogo/protobuf v1.2.1
go: extracting google.golang.org/genproto v0.0.0-20190513181449-d00d292a067c
go: extracting google.golang.org/api v0.5.0
go: extracting github.com/gogo/protobuf v1.2.1
go: downloading github.com/prometheus/procfs v0.0.0-20190507164030-5867b95ac084
go: extracting github.com/prometheus/procfs v0.0.0-20190507164030-5867b95ac084
go: downloading github.com/mailru/easyjson v0.0.0-20180730094502-03f2033d19d5
go: extracting github.com/mailru/easyjson v0.0.0-20180730094502-03f2033d19d5
go: downloading github.com/mattn/go-runewidth v0.0.4
go: extracting github.com/mattn/go-runewidth v0.0.4
go: downloading github.com/Azure/azure-pipeline-go v0.2.1
go: extracting github.com/Azure/azure-pipeline-go v0.2.1
go: downloading github.com/modern-go/concurrent v0.0.0-20180306012644-bacd9c7ef1dd
go: extracting github.com/modern-go/concurrent v0.0.0-20180306012644-bacd9c7ef1dd
go: downloading github.com/mattn/go-ieproxy v0.0.0-20190805055040-f9202b1cfdeb
go: extracting github.com/mattn/go-ieproxy v0.0.0-20190805055040-f9202b1cfdeb
go: downloading github.com/mattn/go-colorable v0.1.4
go: downloading github.com/beorn7/perks v1.0.0
go: downloading github.com/coredns/coredns v1.4.0
go: extracting github.com/beorn7/perks v1.0.0
go: extracting github.com/mattn/go-colorable v0.1.4
go: downloading gopkg.in/ini.v1 v1.57.0
go: downloading github.com/fatih/structs v1.1.0
go: extracting github.com/fatih/structs v1.1.0
go: extracting gopkg.in/ini.v1 v1.57.0
go: downloading golang.org/x/oauth2 v0.0.0-20190402181905-9f3314589c9a
go: downloading github.com/pierrec/lz4 v2.4.0+incompatible
go: extracting golang.org/x/oauth2 v0.0.0-20190402181905-9f3314589c9a
go: downloading git.apache.org/thrift.git v0.13.0
go: extracting github.com/coredns/coredns v1.4.0
go: downloading github.com/philhofer/fwd v1.0.0
go: extracting github.com/philhofer/fwd v1.0.0
go: downloading github.com/rs/cors v1.7.0
go: extracting github.com/rs/cors v1.7.0
go: downloading github.com/google/uuid v1.1.1
go: extracting github.com/google/uuid v1.1.1
go: downloading github.com/golang/protobuf v1.3.2
go: extracting gopkg.in/jcmturner/gokrb5.v7 v7.3.0
go: extracting github.com/golang/protobuf v1.3.2
go: downloading github.com/minio/md5-simd v1.1.0
go: extracting github.com/minio/md5-simd v1.1.0
go: downloading github.com/minio/sha256-simd v0.1.1
go: extracting github.com/minio/sha256-simd v0.1.1
go: downloading github.com/hashicorp/vault/api v1.0.4
go: extracting github.com/hashicorp/vault/api v1.0.4
go: downloading github.com/hashicorp/go-rootcerts v1.0.1
go: extracting github.com/hashicorp/go-rootcerts v1.0.1
go: downloading golang.org/x/time v0.0.0-20190308202827-9d24e82272b4
go: extracting golang.org/x/time v0.0.0-20190308202827-9d24e82272b4
go: downloading gopkg.in/square/go-jose.v2 v2.3.1
go: extracting git.apache.org/thrift.git v0.13.0
go: extracting gopkg.in/square/go-jose.v2 v2.3.1
go: downloading github.com/hashicorp/go-retryablehttp v0.5.4
go: extracting github.com/hashicorp/go-retryablehttp v0.5.4
go: downloading github.com/hashicorp/vault/sdk v0.1.13
go: extracting github.com/hashicorp/vault/sdk v0.1.13
go: downloading github.com/matttproud/golang_protobuf_extensions v1.0.1
go: extracting github.com/matttproud/golang_protobuf_extensions v1.0.1
go: downloading github.com/rcrowley/go-metrics v0.0.0-20190704165056-9c2d0518ed81
go: downloading github.com/eapache/go-resiliency v1.2.0
go: downloading github.com/nats-io/nats.go v1.9.1
go: downloading github.com/go-sql-driver/mysql v1.5.0
go: downloading github.com/beevik/ntp v0.2.0
go: downloading github.com/davecgh/go-spew v1.1.1
go: downloading github.com/eapache/queue v1.1.0
go: downloading github.com/dustin/go-humanize v1.0.0
go: extracting github.com/rcrowley/go-metrics v0.0.0-20190704165056-9c2d0518ed81
go: extracting github.com/davecgh/go-spew v1.1.1
go: extracting github.com/beevik/ntp v0.2.0
go: downloading go.opencensus.io v0.21.0
go: downloading github.com/nsqio/go-nsq v1.0.7
go: downloading github.com/eapache/go-xerial-snappy v0.0.0-20180814174437-776d5712da21
go: downloading github.com/alecthomas/participle v0.2.1
go: extracting github.com/eapache/queue v1.1.0
go: extracting github.com/go-sql-driver/mysql v1.5.0
go: downloading github.com/googleapis/gax-go/v2 v2.0.4
go: extracting github.com/dustin/go-humanize v1.0.0
go: downloading github.com/jcmturner/gofork v0.0.0-20190328161633-dc7c13fece03
go: extracting github.com/eapache/go-resiliency v1.2.0
go: downloading github.com/mitchellh/mapstructure v1.1.2
go: extracting github.com/nats-io/nats.go v1.9.1
go: downloading github.com/xdg/stringprep v1.0.0
go: extracting github.com/nsqio/go-nsq v1.0.7
go: extracting github.com/eapache/go-xerial-snappy v0.0.0-20180814174437-776d5712da21
go: downloading github.com/nats-io/jwt v0.3.2
go: downloading gopkg.in/yaml.v2 v2.2.8
go: extracting go.opencensus.io v0.21.0
go: extracting github.com/jcmturner/gofork v0.0.0-20190328161633-dc7c13fece03
go: extracting github.com/alecthomas/participle v0.2.1
go: extracting github.com/googleapis/gax-go/v2 v2.0.4
go: downloading github.com/hashicorp/go-uuid v1.0.1
go: extracting github.com/mitchellh/mapstructure v1.1.2
go: downloading github.com/hashicorp/errwrap v1.0.0
go: downloading github.com/hashicorp/go-sockaddr v1.0.2
go: extracting github.com/xdg/stringprep v1.0.0
go: downloading github.com/valyala/tcplisten v0.0.0-20161114210144-ceec8f93295a
go: downloading github.com/hashicorp/hcl v1.0.0
go: downloading github.com/klauspost/cpuid v1.3.0
go: extracting github.com/nats-io/jwt v0.3.2
go: downloading github.com/hashicorp/go-multierror v1.0.0
go: extracting github.com/hashicorp/go-uuid v1.0.1
go: extracting github.com/hashicorp/errwrap v1.0.0
go: downloading github.com/hashicorp/golang-lru v0.5.1
go: downloading github.com/golang/snappy v0.0.1
go: extracting gopkg.in/yaml.v2 v2.2.8
go: downloading gopkg.in/jcmturner/goidentity.v3 v3.0.0
go: extracting github.com/valyala/tcplisten v0.0.0-20161114210144-ceec8f93295a
go: downloading github.com/ncw/directio v1.0.5
go: extracting github.com/hashicorp/go-sockaddr v1.0.2
go: downloading github.com/gorilla/handlers v1.4.0
go: extracting github.com/hashicorp/hcl v1.0.0
go: downloading golang.org/x/text v0.3.2
go: downloading github.com/klauspost/readahead v1.3.1
go: extracting github.com/hashicorp/go-multierror v1.0.0
go: extracting github.com/hashicorp/golang-lru v0.5.1
go: downloading github.com/tidwall/gjson v1.3.5
go: extracting github.com/klauspost/cpuid v1.3.0
go: downloading github.com/hashicorp/go-cleanhttp v0.5.1
go: extracting gopkg.in/jcmturner/goidentity.v3 v3.0.0
go: downloading gopkg.in/jcmturner/dnsutils.v1 v1.0.1
go: extracting github.com/ncw/directio v1.0.5
go: extracting github.com/golang/snappy v0.0.1
go: downloading gopkg.in/jcmturner/rpc.v1 v1.1.0
go: extracting github.com/gorilla/handlers v1.4.0
go: downloading gopkg.in/jcmturner/aescts.v1 v1.0.1
go: downloading github.com/rjeczalik/notify v0.9.2
go: extracting github.com/hashicorp/go-cleanhttp v0.5.1
go: extracting github.com/klauspost/readahead v1.3.1
go: downloading github.com/ryanuber/go-glob v1.0.0
go: extracting github.com/tidwall/gjson v1.3.5
go: extracting github.com/rjeczalik/notify v0.9.2
go: downloading github.com/nats-io/nkeys v0.1.3
go: extracting gopkg.in/jcmturner/aescts.v1 v1.0.1
go: downloading github.com/tidwall/pretty v1.0.0
go: extracting gopkg.in/jcmturner/dnsutils.v1 v1.0.1
go: downloading github.com/tidwall/match v1.0.1
go: extracting gopkg.in/jcmturner/rpc.v1 v1.1.0
go: downloading github.com/tidwall/sjson v1.0.4
go: extracting github.com/ryanuber/go-glob v1.0.0
go: extracting github.com/tidwall/pretty v1.0.0
go: extracting github.com/tidwall/sjson v1.0.4
go: extracting github.com/nats-io/nkeys v0.1.3
go: extracting github.com/tidwall/match v1.0.1
go: extracting github.com/pierrec/lz4 v2.4.0+incompatible
go: extracting golang.org/x/text v0.3.2
go: finding github.com/minio/minio-go/v6 v6.0.58-0.20200612001654-a57fec8037ec
go: finding github.com/colinmarc/hdfs/v2 v2.1.1
go: finding github.com/minio/cli v1.22.0
go: finding gopkg.in/jcmturner/gokrb5.v7 v7.3.0
go: finding cloud.google.com/go v0.39.0
go: finding google.golang.org/api v0.5.0
go: finding github.com/dustin/go-humanize v1.0.0
go: finding github.com/json-iterator/go v1.1.10
go: finding github.com/minio/highwayhash v1.0.0
go: finding github.com/Azure/azure-pipeline-go v0.2.1
go: finding github.com/dgrijalva/jwt-go v3.2.0+incompatible
go: finding github.com/gorilla/mux v1.7.0
go: finding github.com/minio/sha256-simd v0.1.1
go: finding github.com/montanaflynn/stats v0.5.0
go: finding github.com/Azure/azure-storage-blob-go v0.8.0
go: finding gopkg.in/jcmturner/dnsutils.v1 v1.0.1
go: finding github.com/jcmturner/gofork v0.0.0-20190328161633-dc7c13fece03
go: finding golang.org/x/crypto v0.0.0-20200604202706-70a84ac30bf9
go: finding github.com/valyala/tcplisten v0.0.0-20161114210144-ceec8f93295a
go: finding github.com/modern-go/concurrent v0.0.0-20180306012644-bacd9c7ef1dd
go: finding github.com/mattn/go-ieproxy v0.0.0-20190805055040-f9202b1cfdeb
go: finding github.com/hashicorp/go-uuid v1.0.1
go: finding golang.org/x/sys v0.0.0-20200610111108-226ff32320da
go: finding github.com/mitchellh/go-homedir v1.1.0
go: finding github.com/rjeczalik/notify v0.9.2
go: finding gopkg.in/ini.v1 v1.57.0
go: finding github.com/fatih/color v1.7.0
go: finding github.com/modern-go/reflect2 v1.0.1
go: finding google.golang.org/genproto v0.0.0-20190513181449-d00d292a067c
go: finding github.com/mattn/go-colorable v0.1.4
go: finding github.com/mattn/go-isatty v0.0.8
go: finding golang.org/x/oauth2 v0.0.0-20190402181905-9f3314589c9a
go: finding github.com/googleapis/gax-go/v2 v2.0.4
go: finding google.golang.org/grpc v1.22.0
go: finding github.com/minio/md5-simd v1.1.0
go: finding golang.org/x/net v0.0.0-20200602114024-627f9648deb9
go: finding github.com/golang/protobuf v1.3.2
go: finding go.opencensus.io v0.21.0
go: finding gopkg.in/jcmturner/aescts.v1 v1.0.1
go: finding github.com/secure-io/sio-go v0.3.0
go: finding github.com/hashicorp/golang-lru v0.5.1
go: finding github.com/shirou/gopsutil v2.20.3-0.20200314133625-53cec6b37e6a+incompatible
go: finding github.com/ncw/directio v1.0.5
go: finding gopkg.in/jcmturner/rpc.v1 v1.1.0
go: finding gopkg.in/jcmturner/goidentity.v3 v3.0.0
go: finding golang.org/x/text v0.3.2
go: finding github.com/klauspost/cpuid v1.3.0
go: finding github.com/cespare/xxhash/v2 v2.1.1
go: finding github.com/cheggaaa/pb v1.0.28
go: finding github.com/mattn/go-runewidth v0.0.4
go: finding github.com/coreos/etcd v3.3.12+incompatible
go: finding github.com/gogo/protobuf v1.2.1
go: finding github.com/dchest/siphash v1.2.1
go: finding github.com/djherbis/atime v1.0.0
go: finding github.com/elazarl/go-bindata-assetfs v1.0.0
go: finding github.com/google/uuid v1.1.1
go: finding github.com/gorilla/handlers v1.4.0
go: finding github.com/gorilla/rpc v1.2.0
go: finding github.com/inconshreveable/go-update v0.0.0-20160112193335-8152e7eb6ccf
go: finding github.com/klauspost/compress v1.10.3
go: finding github.com/klauspost/readahead v1.3.1
go: finding github.com/klauspost/reedsolomon v1.9.9
go: finding github.com/miekg/dns v1.1.8
go: finding github.com/coredns/coredns v1.4.0
go: finding gopkg.in/ldap.v3 v3.0.3
go: finding gopkg.in/asn1-ber.v1 v1.0.0-20181015200546-f715ec2f112d
go: finding github.com/Shopify/sarama v1.24.1
go: finding github.com/davecgh/go-spew v1.1.1
go: finding github.com/eapache/go-resiliency v1.2.0
go: finding github.com/eapache/go-xerial-snappy v0.0.0-20180814174437-776d5712da21
go: finding github.com/golang/snappy v0.0.1
go: finding github.com/eapache/queue v1.1.0
go: finding github.com/pierrec/lz4 v2.4.0+incompatible
go: finding github.com/rcrowley/go-metrics v0.0.0-20190704165056-9c2d0518ed81
go: finding github.com/eclipse/paho.mqtt.golang v1.2.0
go: finding github.com/go-sql-driver/mysql v1.5.0
go: finding github.com/gomodule/redigo v2.0.0+incompatible
go: finding github.com/lib/pq v1.1.1
go: finding github.com/nats-io/nats.go v1.9.1
go: finding github.com/nats-io/jwt v0.3.2
go: finding github.com/nats-io/nkeys v0.1.3
go: finding github.com/nats-io/nuid v1.0.1
go: finding github.com/nats-io/stan.go v0.4.5
go: finding github.com/nsqio/go-nsq v1.0.7
go: finding github.com/pkg/errors v0.8.1
go: finding github.com/streadway/amqp v0.0.0-20190404075320-75d898a42a94
go: finding github.com/xdg/scram v0.0.0-20180814205039-7eeb5667e42c
go: finding github.com/xdg/stringprep v1.0.0
go: finding gopkg.in/olivere/elastic.v5 v5.0.80
go: finding github.com/mailru/easyjson v0.0.0-20180730094502-03f2033d19d5
go: finding github.com/hashicorp/vault/api v1.0.4
go: finding github.com/hashicorp/errwrap v1.0.0
go: finding github.com/hashicorp/go-cleanhttp v0.5.1
go: finding github.com/hashicorp/go-multierror v1.0.0
go: finding github.com/hashicorp/go-retryablehttp v0.5.4
go: finding github.com/hashicorp/go-rootcerts v1.0.1
go: finding github.com/hashicorp/hcl v1.0.0
go: finding github.com/hashicorp/vault/sdk v0.1.13
go: finding github.com/hashicorp/go-sockaddr v1.0.2
go: finding github.com/ryanuber/go-glob v1.0.0
go: finding github.com/mitchellh/mapstructure v1.1.2
go: finding golang.org/x/time v0.0.0-20190308202827-9d24e82272b4
go: finding gopkg.in/square/go-jose.v2 v2.3.1
go: finding github.com/minio/sio v0.2.0
go: finding github.com/beevik/ntp v0.2.0
go: finding github.com/fatih/structs v1.1.0
go: finding gopkg.in/yaml.v2 v2.2.8
go: finding github.com/klauspost/pgzip v1.2.1
go: finding github.com/bcicen/jstream v0.0.0-20190220045926-16c1f8af81c2
go: finding github.com/alecthomas/participle v0.2.1
go: finding github.com/minio/simdjson-go v0.1.5-0.20200303142138-b17fe061ea37
go: finding git.apache.org/thrift.git v0.13.0
go: finding github.com/tidwall/gjson v1.3.5
go: finding github.com/tidwall/match v1.0.1
go: finding github.com/tidwall/pretty v1.0.0
go: finding github.com/tidwall/sjson v1.0.4
go: finding github.com/prometheus/client_golang v0.9.3
go: finding github.com/beorn7/perks v1.0.0
go: finding github.com/prometheus/client_model v0.0.0-20190129233127-fd36f4220a90
go: finding github.com/prometheus/common v0.4.0
go: finding github.com/matttproud/golang_protobuf_extensions v1.0.1
go: finding github.com/prometheus/procfs v0.0.0-20190507164030-5867b95ac084
go: finding github.com/rs/cors v1.7.0
go: finding github.com/tinylib/msgp v1.1.1
go: finding github.com/philhofer/fwd v1.0.0
go: finding github.com/willf/bloom v2.0.3+incompatible
go: finding github.com/spaolacci/murmur3 v1.1.0
go: finding github.com/willf/bitset v1.1.10
# github.com/minio/minio/cmd
cmd/erasure-decode.go:149:27: disk.IsOnline undefined (type io.ReaderAt has no field or method IsOnline)
make: *** [build] Error 2

```

--- a/cmd/erasure-decode.go
+++ b/cmd/erasure-decode.go
@@ -145,9 +145,9 @@ func (p *parallelReader) Read(dst [][]byte) ([][]byte, error) {
 		wg.Add(1)
 		go func(i int) {
 			defer wg.Done()
-			disk := p.readers[i]
-			if disk == nil {
-				// Since disk is nil, trigger another read.
+			rr := p.readers[i]
+			if rr == nil {
+				// Since reader is nil, trigger another read.
 				readTriggerCh <- true
 				return
 			}
@@ -160,7 +160,7 @@ func (p *parallelReader) Read(dst [][]byte) ([][]byte, error) {
 			// For the last shard, the shardsize might be less than previous shard sizes.
 			// Hence the following statement ensures that the buffer size is reset to the right size.
 			p.buf[bufIdx] = p.buf[bufIdx][:p.shardSize]
-			_, err := disk.ReadAt(p.buf[bufIdx], p.offset)
+			_, err := rr.ReadAt(p.buf[bufIdx], p.offset)
 			if err != nil {
 				if _, ok := err.(*errHashMismatch); ok {
 					atomic.StoreInt32(&healRequired, 1)

--- a/cmd/erasure-zones.go
+++ b/cmd/erasure-zones.go
@@ -928,9 +928,18 @@ func (z *erasureZones) listObjects(ctx context.Context, bucket, prefix, marker, 
 // N times until this boolean is 'false'.
 func lexicallySortedEntryZone(zoneEntryChs [][]FileInfoCh, zoneEntries [][]FileInfo, zoneEntriesValid [][]bool) (FileInfo, int, int, bool) {
 	for i, entryChs := range zoneEntryChs {
+		i := i
+		var wg sync.WaitGroup
 		for j := range entryChs {
-			zoneEntries[i][j], zoneEntriesValid[i][j] = entryChs[j].Pop()
+			j := j
+			wg.Add(1)
+			// Pop() entries in parallel for large drive setups.
+			go func() {
+				defer wg.Done()
+				zoneEntries[i][j], zoneEntriesValid[i][j] = entryChs[j].Pop()
+			}()
 		}
+		wg.Wait()
 	}
 
 	var isTruncated = false
@@ -1008,9 +1017,18 @@ func lexicallySortedEntryZone(zoneEntryChs [][]FileInfoCh, zoneEntries [][]FileI
 // N times until this boolean is 'false'.
 func lexicallySortedEntryZoneVersions(zoneEntryChs [][]FileInfoVersionsCh, zoneEntries [][]FileInfoVersions, zoneEntriesValid [][]bool) (FileInfoVersions, int, int, bool) {
 	for i, entryChs := range zoneEntryChs {
+		i := i
+		var wg sync.WaitGroup
 		for j := range entryChs {
-			zoneEntries[i][j], zoneEntriesValid[i][j] = entryChs[j].Pop()
+			j := j
+			wg.Add(1)
+			// Pop() entries in parallel for large drive setups.
+			go func() {
+				defer wg.Done()
+				zoneEntries[i][j], zoneEntriesValid[i][j] = entryChs[j].Pop()
+			}()
 		}
+		wg.Wait()
 	}
 
 	var isTruncated = false
@@ -1141,9 +1159,18 @@ func mergeZonesEntriesCh(zonesEntryChs [][]FileInfoCh, maxKeys int, ndisks int) 
 
 func isTruncatedZones(zoneEntryChs [][]FileInfoCh, zoneEntries [][]FileInfo, zoneEntriesValid [][]bool) bool {
 	for i, entryChs := range zoneEntryChs {
+		i := i
+		var wg sync.WaitGroup
 		for j := range entryChs {
-			zoneEntries[i][j], zoneEntriesValid[i][j] = entryChs[j].Pop()
+			j := j
+			wg.Add(1)
+			// Pop() entries in parallel for large drive setups.
+			go func() {
+				defer wg.Done()
+				zoneEntries[i][j], zoneEntriesValid[i][j] = entryChs[j].Pop()
+			}()
 		}
+		wg.Wait()
 	}
 
 	var isTruncated = false
@@ -1170,9 +1197,18 @@ func isTruncatedZones(zoneEntryChs [][]FileInfoCh, zoneEntries [][]FileInfo, zon
 
 func isTruncatedZonesVersions(zoneEntryChs [][]FileInfoVersionsCh, zoneEntries [][]FileInfoVersions, zoneEntriesValid [][]bool) bool {
 	for i, entryChs := range zoneEntryChs {
+		i := i
+		var wg sync.WaitGroup
 		for j := range entryChs {
-			zoneEntries[i][j], zoneEntriesValid[i][j] = entryChs[j].Pop()
+			j := j
+			wg.Add(1)
+			// Pop() entries in parallel for large drive setups.
+			go func() {
+				defer wg.Done()
+				zoneEntries[i][j], zoneEntriesValid[i][j] = entryChs[j].Pop()
+			}()
 		}
+		wg.Wait()
 	}
 
 	var isTruncated = false

--- a/cmd/object-api-listobjects_test.go
+++ b/cmd/object-api-listobjects_test.go
@@ -573,7 +573,7 @@ func testListObjects(obj ObjectLayer, instanceType string, t1 TestErrHandler) {
 
 	for i, testCase := range testCases {
 		testCase := testCase
-		t.Run(fmt.Sprintf("Test%d-%s", i+1, instanceType), func(t *testing.T) {
+		t.Run(fmt.Sprintf("%s-Test%d", instanceType, i+1), func(t *testing.T) {
 			result, err := obj.ListObjects(context.Background(), testCase.bucketName,
 				testCase.prefix, testCase.marker, testCase.delimeter, int(testCase.maxKeys))
 			if err != nil && testCase.shouldPass {

